iOS_SDK/OneSignalSDK/UnitTests/Shadows/UIApplicationOverrider.m

```diff
@@ -152,21 +152,12 @@ + (void)runBackgroundThreads {
     [appDelegate application:app didRegisterForRemoteNotificationsWithDeviceToken:deviceToken];
 }
 
-// Called on iOS 8+
 - (void) overrideRegisterForRemoteNotifications {
     calledRegisterForRemoteNotifications = true;
     [UIApplicationOverrider helperCallDidRegisterForRemoteNotificationsWithDeviceToken];
 }
 
-// iOS 7
-- (void)overrideRegisterForRemoteNotificationTypes:(UIRemoteNotificationType)types {
-    // Just using this flag to mimic the non-prompted behavoir
-    if (UNUserNotificationCenterOverrider.authorizationStatus != [NSNumber numberWithInteger:UNAuthorizationStatusNotDetermined])
-        [UIApplicationOverrider helperCallDidRegisterForRemoteNotificationsWithDeviceToken];
-}
-
-
-// iOS 8 & 9 Only
+// iOS 9 Only
 - (UIUserNotificationSettings*) overrideCurrentUserNotificationSettings {
```

Comment:
Maybe we should include `iOS 9 Only` still?

Reply:
Hey I am confused by this. The comment above overrideCurrentUserNotificationSettings already says iOS 9 only. should I add iOS 9 only somewhere else?

Reply:
Oh nvm its already there Git was showing comparison weird